### PR TITLE
Allow php8

### PR DESCRIPTION
**Pull request by sadikoff**
Finally, allow php8!

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -37,6 +37,10 @@
         "psr-4": { "Knp\\Bundle\\DisqusBundle\\Tests\\": "tests/" }
     },
 
+    "scripts": {
+        "test": "vendor/bin/phpunit"
+    },
+
```
Comment (weaverryan) [inline composer.json:43]: Is this needed?

File: composer.json [hunk 2]
```diff
@@ -18,15 +18,15 @@
...
-        "symfony/http-client": "^4.4|^5.0"
+        "symfony/http-client": "^4.4 || ^5.3 || ^6.0"
     },
 
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.16",
-        "symfony/framework-bundle": "^4.4 | ^5.0",
-        "symfony/phpunit-bridge": "^5.0",
+        "phpstan/phpstan": "^1.2",
+        "phpunit/phpunit": "^8.5 || ^9.5",
```
Comment (weaverryan) [inline composer.json:28]: We should use `symfony/phpunit-bridge` - 3rd party libs is where that shines, allowing different php versions more easily. But, we could also worry about that later when we centralize things more.

File: phpstan-baseline.neon [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            count: 3
+            path: src/Model/DisqusConfig.php
+        -
+            message: "#\\$string of function base64_encode expects string, string|false given\\.$#"
+            count: 1
+            path: src/Model/DisqusConfig.php
+        -
+            message: "#has parameter \\$(content|parameters|options) with no value type specified in iterable type array\\.$#"
+            count: 4
+            path: src/Model/DisqusConfig.php
```
Comment (weaverryan) [inline phpstan-baseline.neon:50]: These aren't things we can more properly fix?

File: phpunit.xml.dist [hunk 4]
```diff
@@ -1,36 +1,10 @@
...
-
-
+<phpunit colors="true" bootstrap="vendor/autoload.php">
     <testsuites>
-        <testsuite name="all">
-            <directory>./tests</directory>
-        </testsuite>
-        <testsuite name="integration">
-            <directory>./tests/IntegrationTests</directory>
+        <testsuite name="KnpPaginatorBundle Test Suite">
```
Comment (weaverryan) [inline phpunit.xml.dist:6]:
```suggestion
        <testsuite name="Test Suite">
```

No reason to include the bundle name anyways ;)

File: phpstan-baseline.neon (hunk 3 above)
Comment (sadikoff) [reply phpstan-baseline.neon:50]: no that requires some complex PHPDoc blocks to show what is inside the array, like `array<string|array|int>` or maybe `array<mixed>`

Comment (weaverryan) [reply phpstan-baseline.neon:50]: Fair enough - it's hard because the signatures of the code in that file are so ugly / "free form".

**Comment by weaverryan** [comment]
Thank you Vladimir!

**Pull request merged**